File: java/2639.java
import java.util.ArrayList;
import java.util.List;

class Solution {

    private int len(int num) {
        return String.valueOf(num).length();
    }
    public int[] findColumnWidth(int[][] grid) {
        int[] widths = new int[grid[0].length];

        for (int i = 0;i<grid.length;i++) {
            for (int j=0;j<grid[0].length;j++) {
                if (len(grid[i][j]) > widths[j]) widths[j] = len(grid[i][j]);
            }
        }

        return widths;
    }
}

File: kt/2639.kt
class Solution {
    fun Int.len(): Int {
        return this.toString().length
    }

    fun findColumnWidth(grid: Array<IntArray>): IntArray {
        val widths = IntArray(grid[0].size)
        for (i in grid.indices) {
            for (j in grid[0].indices) {
                if (grid[i][j].len() > widths[j]) widths[j] = grid[i][j].len()
            }
        }
        return widths
    }
}